crates/api_crud/src/site/mod.rs

```diff
@@ -40,12 +40,19 @@ pub fn application_question_check(
   }
 }
 
+fn not_zero(val: Option<i32>) -> Option<i32> {
+  match val {
+    Some(0) => None,
+    v => v,
+  }
+}
```

Comment:
It'd be good to coerce a maximum value also, to None

Reply:
What would you use as maximum? Anyway this can only be set by admins, so I dont see any need for strict verification. Its enough to ensure that it doesnt cause crashes/bugs.

Reply:
Like 10k or something, but its not a big deal.